src/less/success-message/base/success-message.less

```diff
@@ -0,0 +1,53 @@
+.success-message--full {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    position: relative;
+
+    .fake-btn,
+    .btn {
+        bottom: 0;
+        left: 0;
+        position: absolute;
+        width: 100%;
+    }
+}
+
+.success-message {
+    background-color: @success-message-background-color;
+    box-sizing: border-box;
+    padding: 0 9%;
+    text-align: center;
+    width: 100%;
+
+    .fake-btn,
```

Comment:
Rather than having a dependency on the entire button module, should we just create `success-message__button` and `success-message__fake-button` instead? Then bring in any base button styles via mixins? This goes back to the question of nesting vs not-nesting. I think my preference in this particular case is not to nest.

Reply:
👍 will do.  
(FYI this will be the 5th place we use the btn-base mixin. So the 5th time that code  is duplicated, for teams that bring in the whole skin package.)

Reply:
Yeah, that is a concern, and the drawback of not nesting. If we were using the full array of buttons variants (primary, secondary, small, large, etc), then I would say to just nest. Hopefully the base mixin is quite small...